backend/api/tests/test_.py

```diff
@@ -0,0 +1,220 @@
+#     def test_invalid_name(self):
+#         sample_name = f"{'abc'*100}"
+#         with self.assertRaises(ValidationError):
+#             with open(file=self.file_path, mode="rb") as infile:
+#                 file = SimpleUploadedFile(self.file_path, infile.read())
+#                 item = Item.objects.create(
+#                     image=file,
+#                     name=sample_name,
+#                     stock=1,
+#                     price=1.0,
+#                     description="Item Description",
+#                 )
+#                 item.full_clean()
+        
+#     def test_invalid_stock(self):
+#         sample_stock = -1
+#         with self.assertRaises(ValidationError):
+#             with open(file=self.file_path, mode="rb") as infile:
+#                 file = SimpleUploadedFile(self.file_path, infile.read())
+#                 item = Item.objects.create(
+#                     image=file,
+#                     name="Item",
+#                     stock=sample_stock,
+#                     price=1.0,
+#                     description="Item Description",
+#                 )
+#                 item.full_clean()
+        
+#     def test_invalid_price(self):
+#         sample_price = -1.0
+#         with self.assertRaises(ValidationError):
+#             with open(file=self.file_path, mode="rb") as infile:
+#                 file = SimpleUploadedFile(self.file_path, infile.read())
+#                 item = Item.objects.create(
+#                     image=file,
+#                     name="Item",
+#                     stock=1,
+#                     price=sample_price,
+#                     description="Item Description",
+#                 )
+#                 item.full_clean()
+        
+#         sample_price = 1.123
+#         with self.assertRaises(ValidationError):
+#             with open(file=self.file_path, mode="rb") as infile:
+#                 file = SimpleUploadedFile(self.file_path, infile.read())
+#                 item = Item.objects.create(
+#                     image=file,
+#                     name="Item",
+#                     stock=1,
+#                     price=sample_price,
+#                     description="Item Description",
+#                 )
+#                 item.full_clean()
+        
+#         sample_price = 1234567.123456
+#         with self.assertRaises(ValidationError):
+#             with open(file=self.file_path, mode="rb") as infile:
+#                 file = SimpleUploadedFile(self.file_path, infile.read())
+#                 item = Item.objects.create(
+#                     image=file,
+#                     name="Item",
+#                     stock=1,
+#                     price=sample_price,
+#                     description="Item Description",
+#                 )
+#                 item.full_clean()
+
+class Test_User_Model(TestCase):
+
+    DEFAULT_NAME = "Store"
+    DEFAULT_ADDRESS = "123 Lane"
+    DEFAULT_CATEGORY = Category.FOOD
+   
+    def setup(self):
```

Comment:
In case you're looking for the bug in this, it should be `def setUp(self):`

Reply:
@MattKersey  Bruh....thank you so much for the help 😭